### ISSUE TITLE
pretrain_transformer.pth

### ISSUE THREAD

**Issue by FocueAI** [csguoh/LEMMA#23]
![image](https://github.com/csguoh/LEMMA/assets/49752023/47dad5f0-9c91-4fd2-9149-70c24a1ae197)
Hi! 
I am very interested in this project. 
Can you tell me what project does the weight pretrain_transformer.pth come from?

**Comment by FocueAI** [comment]
Also, have you conducted an experiment using a Chinese character set? How should pre train_transformer. pth be replaced?

**Comment by csguoh** [comment]
Hi, thanks for your interest in this work. The `pretrained_transformer` is used for calculating the text position loss, which is proposed[ in this work](https://openaccess.thecvf.com/content/CVPR2021/html/Chen_Scene_Text_Telescope_Text-Focused_Scene_Image_Super-Resolution_CVPR_2021_paper.html). Since there is few work in Chinese text dataset, we are not able to conduct experiments on Chinese characters. If you want to train a Chinese character STISR models, you may need to re-train the `pretrained_transformer` for loss function, and also retrain  the LEMMA model for Chinese character.

**Comment by FocueAI** [comment]
I really feel your answer!
If I want to retrain the pretrained transformer, can you provide me with the corresponding training code? Looking forward to your answer and help again!

**Comment by FocueAI** [comment]
in addition [in this work](https://openaccess.thecvf.com/content/CVPR2021/html/Chen_Scene_Text_Telescope_Text-Focused_Scene_Image_Super-Resolution_CVPR_2021_paper.html)  [Corresponding code](https://github.com/FudanVI/FudanOCR/tree/main/scene-text-telescope)，I still can't find the training code corresponding to pretrained_transformer

**Comment by csguoh** [comment]
Hi, you may contact the author of STT for the training code of pretrained_transformer.pth.
